packages/base/components/Content/ContentTagsDropdown.vue

```diff
@@ -128,13 +118,15 @@ const clickOutsideConfig = ref({
 </script>
 
 <template>
-  <div>
+  <ContentFeaturedTags :tags="tags" :selected-tags="selectedTags" />
```

Comment:
The featured tags are not rendered within the dropdown, so should not be nested in this component. Consider moving the tag retrieval logic up to a new parent layer which would render both the featured tags and dropdown child components as siblings.